python/cuml/dask/naive_bayes/naive_bayes.py

```diff
@@ -217,7 +217,10 @@ def _get_part(parts, idx):
     def _get_size(arrs):
         return arrs.shape[0]
 
-    def predict(self, X, delayed=True, parallelism=5):
+    def predict(self, X):
+        # TODO: Once cupy sparse arrays are fully supported
+        #  underneath Dask arrays,
+        # this can extend DelayedPredictionMixin.
```

Comment:
Can you please provide context as to what is still needed? Any open issues or PRs? Admittedly these likely won't get solved near term, but it would be good to keep track of all of them somewhere?